dev/tests/integration/framework/Magento/TestFramework/TestCase/AbstractController.php

```diff
@@ -15,6 +15,7 @@
 
 /**
  * @SuppressWarnings(PHPMD.NumberOfChildren)
+ * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
```

Comment:
Yeah those changes seem good to me still not 100% sure why that change fixes the static check, but I can live with not knowing that currently 👍